tests/data/serialization/ndjson/test_dicom.py

```diff
@@ -0,0 +1,183 @@
+    'classifications': [],
+    'dataRow': {
+        'id': 'test-uid'
+    },
+    'name':
+        'dicom_polyline',
+    'groupKey':
+        'axial',
+    'segments': [{
+        'keyframes': [{
+            'frame':
+                2,
+            'line': [
+                {
+                    'x': 680.0,
+                    'y': 100.0
+                },
+                {
+                    'x': 100.0,
+                    'y': 190.0
+                },
+                {
+                    'x': 190.0,
+                    'y': 220.0
+                },
+            ]
+        }]
+    }],
+}
+
+instance_uri_1 = 'https://storage.labelbox.com/cjhfn5y6s0pk507024nz1ocys%2F1d60856c-59b7-3060-2754-83f7e93e0d01-1?Expires=1666901963361&KeyName=labelbox-assets-key-3&Signature=t-2s2DB4YjFuWEFak0wxYqfBfZA'
+instance_uri_5 = 'https://storage.labelbox.com/cjhfn5y6s0pk507024nz1ocys1%2F1d60856c-59b7-3060-2754-83f7e93e0d01-1?Expires=1666901963361&KeyName=labelbox-assets-key-3&Signature=t-2s2DB4YjFuWEFak0wxYqfBfZA'
+frames = [
+    lb_types.MaskFrame(index=1, instance_uri=instance_uri_1),
+    lb_types.MaskFrame(index=5, instance_uri=instance_uri_5)
+]
+instances = [
+    lb_types.MaskInstance(color_rgb=(0, 0, 255), name="mask1"),
+    lb_types.MaskInstance(color_rgb=(0, 255, 0), name="mask2"),
+    lb_types.MaskInstance(color_rgb=(255, 0, 0), name="mask3")
+]
+
+dicom_mask_annotation = lb_types.DICOMMaskAnnotation(
+    name="dicom_mask",
+    group_key=lb_types.GroupKey.AXIAL,
+    frames=frames,
+    instances=instances)
+
+video_mask_annotation = lb_types.VideoMaskAnnotation(name="video_mask",
+                                                     frames=frames,
+                                                     instances=instances)
+
+dicom_mask_label = lb_types.Label(data=lb_types.DicomData(uid="test-uid"),
+                                  annotations=[dicom_mask_annotation])
+
+video_mask_annotation_ndjson = {
+    'dataRow': {
+        'id': 'test-uid'
+    },
+    'name': 'video_mask',
+    'masks': {
+        'frames': [{
+            'index': 1,
+            'instanceURI': instance_uri_1
+        }, {
+            'index': 5,
+            'instanceURI': instance_uri_5
+        }],
+        'instances': [
+            {
+                'colorRGB': (0, 0, 255),
+                'name': 'mask1'
+            },
+            {
+                'colorRGB': (0, 255, 0),
+                'name': 'mask2'
+            },
+            {
+                'colorRGB': (255, 0, 0),
+                'name': 'mask3'
+            },
+        ]
+    },
+}
+
+video_mask_label = lb_types.Label(data=lb_types.VideoData(uid="test-uid"),
+                                  annotations=[video_mask_annotation])
+
+dicom_mask_annotation_ndjson = copy(video_mask_annotation_ndjson)
+dicom_mask_annotation_ndjson['groupKey'] = 'axial'
+dicom_mask_annotation_ndjson['name'] = 'dicom_mask'
+
+
+def test_serialize_dicom_polyline_annotation():
```

Comment:
I really prefer that we start comparing class types and actual attributes instead of serialize/deserialize stuff. I have discovered some tests passed while the actual object we deserialized into was the wrong type (I think our test fixture was wrong in that case).
Do we agree on this? Perhaps do not need to do for this PR if too much work

Reply:
I agree with you, comparing python objects instead of only using a json to deserialize then serialize catches more problems.